framework-plugins/lisk-framework-dashboard-plugin/src/ui/styles/_constants.scss

```diff
@@ -82,3 +82,19 @@ $breakpoint-down: (
 		@content;
 	}
 }
+
+@mixin lisk-scrollbar {
+	scrollbar-width: 14px;
+	scrollbar-color: #c5cfe4;
```

Comment:
maybe define them as `$color-xx` and refer here and &::-webkit-scrollbar-thumb background?